nifi-nar-bundles/nifi-jslt-bundle/nifi-jslt-processors/src/main/java/org/apache/nifi/processors/jslt/JSLTTransformJSON.java

```diff
@@ -122,11 +139,18 @@ public class JSLTTransformJSON extends AbstractProcessor {
     private static final List<PropertyDescriptor> descriptors;
     private static final Set<Relationship> relationships;
     private static final ObjectMapper jsonObjectMapper = new ObjectMapper();
+    private static final ObjectMapper jsonObjectMapperPerObject = new ObjectMapper();
```

Comment:
Is there a reason for this second ObjectMapper?

Reply:
Not sure, I was trying different things when getting the per-object part working. I will remove it and see what happens :)